README.md

```diff
@@ -1,33 +1,49 @@
+## Table of Contents
 
-- [How to deploy](#how-to-deploy): This is the recommended installation option. Use this option if you want to experience the app and the code.
+- [What Is Included in This Project](#what-is-included-within-this-project): A description of the contents of this repository.
 
-- [Installing the app using an Unlocked Package](#2c-installing-the-app-using-an-unlocked-package): This option allows anybody to experience the sample app without installing a local development environment.
+- [How to Deploy](#how-to-deploy): This is the recommended installation option. Use this option if you want to see the app and the code in action.
 
-- [Description of Files and Directories](#description-of-files-and-directories): A description of all the content of this repository.
+- [How to Add New Fields for Classification](#how-to-add-new-fields-for-classification): A guide to specify additional fields for classification.
 
-## What is included within this project
+## What Is Included in This Project
 
-1. A lightning flow already setup with multiple steps to create a support case.
-2. A first screen where the user can enter a Subject and a Description for their case. This screen will predict values for the `Case Reason` field as it is a Standard Salesforce Case field.
-3. A second screen where documentation will be suggested to the customer based on the text entered in the case Subject and Description.
-4. A final screen where the customer can review their case information if they were not able to solve it.
-5. A confirmation screen that the case was created.
+This project contains two flows: a Recommended Flow and a Demo flow. Both are lightning flows already setup with multiple steps to create a support case.
 
-## How to deploy
+### Recommended Flow:
+
+The Recommended Flow contains the following screens:
+
+1. A first screen where the user can enter a Subject and a Description for their case and can see the strength of this Description using the Description Strength Indicator.
+2. A second screen where the user can find predictions to help classify their case. This screen will predict values for the Case Priority, Case Type, and Case Reason fields as these are Standard Salesforce Case fields.
+3. A third screen where documents will be suggested to the user based on the text entered in the case Subject and Description. The user can read more about each document suggestion with the help of the Quickview and can also leave their feedback on each document.
+4. A confirmation screen that the case was created.
+
+### Demo Flow:
+
+The Demo Flow contains the following screens:
+
+1. A first screen where the user can enter a Subject and a Description for their case and can see the strength of this Description, using the Description Strength Indicator, as well as the predictions to help classify their case. The user can see the predictions for the Case Priority, Case Type, and Case Reason fields as they type in the Subject and Description inputs.
```

Comment:
See "How to Add New Fields for Classification" section to learn how to modify these.